SAMtags.tex

```diff
@@ -58,6 +58,7 @@ \section{Standard tags}
   {\tt BC} & Z & Barcode sequence \\
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
   {\tt CC} & Z & Reference name of the next hit \\
+  {\tt CG} & B,I & BAM-only tag to store the real {\sf CIGAR} if it contains $>$65535 operations\\
```

Comment:
Sure, point taken. How about "Intended to store real CIGAR if it contains >65535 operations"? I would like to choose wording such that this tag won't be abused to frequently store normal CIGARs.